src/easyscience/global_object/global_object.py

```diff
@@ -36,6 +36,8 @@ def __init__(self):
         # Map. This is the conduit database between all global object species
         self.map: Map = self.__map
 
+        self.update_id_iterator = 0
```

Comment:
please describe this variable, seeing as every other self variable is properly documented.